specification/search/data-plane/Azure.Search/preview/2019-05-06-preview/searchservice.json

```diff
@@ -7802,6 +7778,33 @@
       },
       "description": "The language codes supported for input by OcrSkill."
     },
+    "OcrSkillLineEnding": {
+      "type": "string",
+      "enum": [
+        "Space",
+        "CarriageReturn",
+        "LineFeed"
+      ],
+      "x-ms-enum": {
+        "name": "LineEndingOption",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "Space",
+            "description": "Space, which is the default value for line endings"
+          },
+          {
+            "value": "CarriageReturn",
+            "description": "Carriage return, represented by \r\n"
```

Comment:
You'll need to escape the backslashes with another backslash, i.e. `\\r\\n`. Same below for `\\n`.

Reply:
Thanks, good catch.  Fixed in https://github.com/Azure/azure-rest-api-specs/pull/9604/commits/1d36c656614ae3c36a92e9c72b515a06bc787c3d.